int32 num1
int32 num2
char drive
